src/models/stores/sorted-documents.ts

```diff
@@ -204,7 +204,7 @@ export class SortedDocuments {
         createdAt: doc.createdAt,
         title: doc.title,
         properties: undefined,
-        tileTypes: Array.from(doc.content?.tileTypes || []),
+        tools: Array.from(doc.content?.tileTypes || []),
```

Comment:
Probably this `content?.tileTypes` needs to be updated to include the "Sparrow" type too, and it might make sense to rename it to `tools`.

Reply:
Instead of updating the doc.content.tileTypes method, I added some code to compute the tools array from both the tileTypes and the annotations. Let me know what you think.